style.css

```diff
@@ -7,24 +7,23 @@ body {
 }
 
 nav {
-  position: ;
+  position: absolute;
 }
 
 header {
   justify-content: center;
   border: 1px solid cyan;
```

Comment:
- [ ] Please remove all of the borders that are not part of the design once you have finished using them to layout the elements

Reply:
Yes, sorry, I was trying to find a way to show the problems I was getting with the flexbox.